src/forms/button/Button.tsx

```diff
@@ -44,10 +45,14 @@ export enum ButtonSize {
 // TODO: add loading support
 
 export class Button extends React.PureComponent<ButtonProps> {
+    static defaultProps = {
+        customBtn: false,
+    };
+
     private static getClassNames(props: ButtonProps): (string | undefined)[] {
         return [
-            "btn",
-            props.icon ? "btn-icon" : undefined,
+            props.customBtn ? undefined : "btn",
```

Comment:
You can change this to something like : props.customBtn  && "btn"

Reply:
I agree.  Can be shortened using `&&`.

Reply:
Done